dashboard/src/components/widget/loader/che-loader-animation-panel.styl

```diff
@@ -0,0 +1,4 @@
+.loader .animation-panel
+  border 3px solid #979797
+  background-color #fff
```

Comment:
Can we use standard colors from  che-colors.json?
https://github.com/eclipse/che/blob/master/dashboard/src/app/colors/che-colors.json#L7
https://github.com/eclipse/che/blob/master/dashboard/src/app/colors/che-colors.json#L19